doitlive/__init__.py

```diff
@@ -221,13 +220,15 @@ def echo(message=None, file=None, nl=True, err=False, color=None, carriage_retur
 PERCENT = TermString(u'%')
 NEW_LINE = TermString(u'\n')
 
+
```

Comment:
We explicitly ignore the 2-line rule in our flake8 config because it results in unnecessary whitespace. But hey, if it makes you happier, then I'm good with it.